### DISCUSSION TITLE
Will the Vectara vector store be implemented?

### DISCUSSION THREAD

**Discussion by 3eif** [langflow-ai/langflow#897] [General]
I was wondering if the Vectara vector store is implemented or not based on this file: https://github.com/logspace-ai/langflow/blob/dev/src/backend/langflow/components/vectorstores/Vectara.py

And if it's not, is there a plan for it to be implemented?

**Comment by rodrigosnader** [comment]
Done.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
